### DISCUSSION TITLE
I can't login

### DISCUSSION THREAD

**Discussion by Hex3767** [ppy/osu#15858] [Q&A] [answered]
I'm having a bit of a problem with osu!lazer on android. I installed the game from taptap. i tried to login so as to download beatmaps, the game says "attempting to connect" for a brief second and then nothing happened

>

**Comment by bdach** [comment, marked as answer]
i don't know what "taptap" is but it's not an official or supported source. we don't publish there, the APK may as well be backdoored for all i know. please only use official releases from this github page. you may also want to change your password.

**Reply by Hex3767** [reply]
I downloaded lazer again from the official github, although, a new problem occured. I can't type in-game, which means i still can't login

**Reply by nahkd123** [reply]
> I downloaded lazer again from the official github, although, a new problem occured. I can't type in-game, which means i still can't login

Well then that would be an issue with android keyboard again (which is already discussed in https://github.com/ppy/osu/discussions/15575). You can try different keyboard and see if it is working or not.

**Comment by Loreos7** [comment]
You should download apk from here https://github.com/ppy/osu/releases/tag/2021.1127.0 (latest for now) and try this again. Should work.
